src/workloads/query/QueryStats.yml

```diff
@@ -36,13 +36,13 @@ ActorTemplates:
           SleepBefore: 5 seconds
           Database: admin
           Operations:
-          - OperationMetricsName: TelemetryWithRedactionMetrics
+          - OperationMetricsName: QueryStatsWithRedactionMetrics
```

Comment:
Does this mean their metrics are combined in the output accidentally? I don't see the actor below overriding this value in the template? I am realizing I'm not actually sure what "OperationMetricsName" does...

Reply:
Oh that's good to know! Thanks! Either way is fine with me. Should be hard to mis-interpret with either title, but slightly more clear with the customized name.

Reply:
Sounds good! I might just have it say `QueryStatsMetrics` so that it doesn't get confusing, since right now I realized for the case without HMAC we have "WithoutHMAC" and "WithHMACMetrics" which contradict each other

Reply:
I like it.